kima/chain.json

```diff
@@ -0,0 +1,119 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "kima",
+  "status": "live",
+  "website": "https://kima.network/",
+  "network_type": "mainnet",
+  "pretty_name": "Kima",
+  "chain_type": "cosmos",
+  "chain_id": "kima",
+  "bech32_prefix": "kima",
```

Comment:
Please register this prefix
https://github.com/satoshilabs/slips/blob/master/slip-0173.md

Reply:
Registered: https://github.com/satoshilabs/slips/pull/1790